ExomeDepth/make_BEDdetail.py

```diff
@@ -100,69 +139,99 @@ def slice_vcf(args, merge_dic):
                 ccn = float(record.genotype(sampleid)['CCN'])
                 bf = float(record.genotype(sampleid)['BF'])
 
-                if (totalcalls < args.totalcallsqc_min or
-                    totalcalls > args.totalcallsqc_max or
-                    correl < args.correlqc or
-                    deldupratio < args.deldupratioqc_min or
-                    deldupratio > args.deldupratioqc_max):
-                        excluded_samples_file.write("Sample {sampleid} run {runid} is excluded because not all QC are above threshold\n".format(sampleid=sampleid, runid=runid))
-                        if sampletype == "child":
-                            children -= 1
-                        elif sampletype == "parent":
-                            parents -= 1
-                        break
+                if(totalcalls < args.totalcallsqc_min or
+                   totalcalls > args.totalcallsqc_max or
+                   correl < args.correlqc or
+                   deldupratio < args.deldupratioqc_min or
+                   deldupratio > args.deldupratioqc_max):
+                    excluded_samples_file.write(
+                        "Sample {sampleid} run {runid} is excluded because not all QC are above threshold\n".format(
+                            sampleid=sampleid,
+                            runid=runid
+                        )
+                    )
```

Comment:
```python
if(
    totalcalls < args.totalcallsqc_min or
    totalcalls > args.totalcallsqc_max or
    correl < args.correlqc or
    deldupratio < args.deldupratioqc_min or
    deldupratio > args.deldupratioqc_max
):
    excluded_samples_file.write(
        "Sample {sampleid} run {runid} is excluded because not all QC are above threshold\n".format(
            sampleid=sampleid,
            runid=runid
     )
)

```